main.go

```diff
@@ -14,24 +14,32 @@ import (
 
 func main() {
 	home, err := os.UserHomeDir()
+
 	if err != nil {
 		log.Fatal(err)
 	}
+
 	defaultConfigPath := fmt.Sprintf("%s/.config/gobee/config.yaml", home)
 	contents, err := ioutil.ReadFile(defaultConfigPath)
 	if err != nil {
-		log.Fatalf("%s\nHave you created a Gobee config file at %s?", err, defaultConfigPath)
+		log.Fatalf("Failed to fin Gobee config %s\n", err)
 	}
 
 	var cfg config.Config
 	err = yaml.Unmarshal(contents, &cfg)
 	if err != nil {
-		log.Fatalf("cmd.Run() failed with %s\n", err)
+		log.Fatalf("Failed to Unmarshal YAMl. %s\n", err)
```

Comment:
Not sure if people will understand this terminology. Perhaps `parse` would fit better :face_with_monocle:.